docs/run-instance/example-setups/de_tomcat-10-ubuntu_22_04.md

```diff
@@ -0,0 +1,406 @@
+Passe den Test entsprechend deiner heruntergeladenen OSM-Datei an.
+
+```shell
+# Teste eine Route von A nach B in Andorra
+> curl -X POST \
+  'http://localhost:8080/ors/v2/directions/driving-car' \
+  -H 'Content-Type: application/json; charset=utf-8' \
+  -H 'Accept: application/json, application/geo+json, application/gpx+xml, img/png; charset=utf-8' \
+  -d '{"coordinates":[[1.5036892890930176, 42.4972256361276],[1.6298711299896242, 42.57831077271361]]}'
+
+# Teste eine Isochrone von einem Punkt in Andorra
+> curl -X POST \
+  'http://localhost:8080/ors/v2/isochrones/driving-car' \
+  -H 'Content-Type: application/json; charset=utf-8' \
+  -H 'Accept: application/json, application/geo+json, application/gpx+xml, img/png; charset=utf-8' \
+  -d '{"locations":[[1.5036892890930176, 42.4972256361276]],"range":[300]}'
+```
+
+## Optional: Pre-Build Graphen updaten
+
+Falls du nur die vorhandenen Graphen durch neue vorgefertigte ersetzen möchtest, folge diesen Schritten:
+
+1. Lade die neuen Graphen von der entsprechenden Quelle herunter.
+   Diese liegen in der Regel in einer `zip-` oder `tar.xz`-Datei vor.
+2. Entpacke die neuen Graphen und stelle sicher, dass sie das richtige Format haben.
+
+```shell
+# Im Fall einer tar.xz-Datei
+> tar -xvf new_graphs.tar.xz
+# Im Fall einer ZIP-Datei kannst du den unzip-Befehl verwenden
+> unzip new_graphs.zip
+```
+
+Die entpackten Graphen haben eine ähnliche Ordnerstruktur wie diese:
+
+```shell
+new_graphs/
+├── driving-car
+└── cycling-regular
+```
+
+3. Leere das alte Graphen-Verzeichnis `/opt/openrouteservice/graphs/`:
+
+```shell
+> mv /opt/openrouteservice/graphs /opt/openrouteservice/graphs_old
+> mkdir /opt/openrouteservice/graphs
+```
+
+4. Verschiebe die neuen Graphen in das Verzeichnis `/opt/openrouteservice/graphs/`.
+   Die neue Graphen-Ordnerstruktur sollte wie folgt aussehen:
+
+```shell
+/opt/openrouteservice/graphs/
+├── driving-car
+└── cycling-regular
+```
+
+5. Setze die [korrekten Berechtigungen](#openrouteservice-ausführen) für das Tomcat-Verzeichnis und starte den
+   Tomcat-Dienst neu.
+6. Führe die [Beispielanfragen](#beispielanfragen) aus, um zu überprüfen, ob die neuen Graphen korrekt funktionieren.
+
+## Optional: openrouteservice aktualisieren
+
+Falls du openrouteservice auf eine neue Version aktualisieren möchtest, kannst du die folgenden Schritte ausführen:
+
+1. Stoppe den Tomcat-Dienst.
+
+```shell
+> sudo systemctl stop openrouteservice.service
+```
+
+2. Lade die neue [WAR-Datei](#das-openrouteservice-war-file-herunterladen) für die gewünschte Version herunter.
+3. Leere das alte Webapps-Verzeichnis `/opt/tomcat/webapps/` und verschiebe die neue `WAR-Datei` in dieses Verzeichnis.
+4. Passe die [openrouteservice-Konfiguration](#openrouteservice-konfigurieren) in der Datei `setenv.sh` an.
+5. [Setze die korrekten Berechtigungen](#openrouteservice-ausführen) für das Tomcat-Verzeichnis und starte den
```

Comment:
```suggestion
5. [Setze die korrekten Berechtigungen](#setze-die-notwendigen-berechtigungen-fur-den-tomcat-10-benutzer) für das Tomcat-Verzeichnis und starte den
```

Reply:
The suggested link does not point to a valid anchor.